docs/ancestry.rst

```diff
@@ -748,10 +731,92 @@ rate along the genome can be combined in ``msprime``.
     import msprime
     from IPython.display import SVG
 
-.. todo:: Examples of small simulations using GC on its own
-    and mixed with recombination. Ideally we'd start with an
-    example with one GC event where we could explain what happened.
-    This might be tricky to finesse.
+Gene conversion events are defined by two parameters. 
+For discrete genomes these are:
```

Comment:
I would phrase this slightly differently to take the emphasis off discrete/continous genomes:

> Gene conversion events are defined by two parameters: the rate at which gene conversion events are initiated and the distribution of tract lengths. In the default case of discrete genome coordinates, tract lengths are drawn from a geometric distribution with mean ``gene_conversion_tract_length`` (which must be at least 1). Note that if we specify a tract length of 1, then all gene conversion tracts will have length exactly 1.

Reply:
Yes, that is better. Changed it.